From dc5347ace91dee331c9375540d65301c1af29706 Mon Sep 17 00:00:00 2001
From: Harsh J <harsh@cloudera.com>
Date: Mon, 30 Mar 2015 15:21:18 +0530
Subject: [PATCH 1866/2848] HDFS-4396. Add START_MSG/SHUTDOWN_MSG for ZKFC.
 Contributed by Liang Xie.

(cherry picked from commit ae3e8c61ff4c926ef3e71c782433ed9764d21478)
(cherry picked from commit c58357939fecf797d9556f70d434edba81681f6f)

Change-Id: I265acfe3960387aa6ff69798d2ee6cc495cd8f1d
---
 .../hadoop/hdfs/tools/DFSZKFailoverController.java |    2 ++
 1 file changed, 2 insertions(+)

diff --git a/hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/tools/DFSZKFailoverController.java b/hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/tools/DFSZKFailoverController.java
index 45561f4..f125a27 100644
--- a/hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/tools/DFSZKFailoverController.java
+++ b/hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/tools/DFSZKFailoverController.java
@@ -176,6 +176,8 @@ protected String getScopeInsideParentNode() {
 
   public static void main(String args[])
       throws Exception {
+    StringUtils.startupShutdownMessage(DFSZKFailoverController.class,
+        args, LOG);
     if (DFSUtil.parseHelpArgument(args, 
         ZKFailoverController.USAGE, System.out, true)) {
       System.exit(0);
-- 
1.7.9.5

